index.html

```diff
@@ -29,9 +30,16 @@
       </div>
 
       <div class="col-lg-6 col-md-12">

```

Comment:
it is possible https://github.com/SMH110/widgets/pull/52/commits/21fa303195e1b42545fb519e4086f258be202e1b